certora/README.md

```diff
@@ -0,0 +1,36 @@
+# Morpho token contract formal verification
+
+This folder contains the [CVL](https://docs.certora.com/en/latest/docs/cvl/index.html) specification and verification setup for the [MorphoTokenEthereum](../src/MorphoTokenEthereum.sol) and  [MorphoTokenOptimism](../src/MorphoTokenOptimism.sol) contracts.
+
+## Getting started
+
+This project depends on [Solidity](https://soliditylang.org/) which is required for running the verification.
+The compiler binary should be available in the path:
+
+- `solc-0.8.27` for the solidity compiler version `0.8.27`.
+
+To verify a specification, run the command `certoraRun Spec.conf` where `Spec.conf` is the configuration file of the matching CVL specification.
+Configuration files are available in [`certora/confs`](confs).
+Please ensure that `CERTORAKEY` is set up in your environment.
+
+## Overview
+
+These Morpho token contracts replace the legacy version and add support for delegation of voting power, upgradeability and cross-chain interactions.
+Due to the upgradeable nature of the new Morpho token, the contract isn't properly immutable.
+Instead, we verify that the implementation doesn't perform delegate calls.
```

Comment:
```suggestion
The contract is upgradeable, but we can verify that the implementation is immutable.
```

Reply:
Fixed in https://github.com/morpho-org/morpho-token-upgradeable/pull/90/commits/03434a7602ec685723745bf2d238237644451934.